### DISCUSSION TITLE
Welcome to result Discussions!

### DISCUSSION THREAD

**Discussion by wbolster** [rustedpy/result#86] [Announcements]
## 👋 Welcome!

As an experiment, we’ve opened Discussions as a place to connect with other members of our community. You can:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by rvente** [comment]
Hello  :wave: Thanks for this.

**Comment by mateus-cortiano** [comment]
Hello guys!

I really loved this Result type and currently using in some personal projects of mine. Actually, I can't not use it anymore because I like it so much.

I was fooling around trying to implement the Option type of rust with some kind of success. If you want to check it out and maybe add to the rustedpy repo send me a message!

**Reply by francium** [reply]
@mateus-cortiano 

Please feel free to open a new discussion thread here and share your Option type and examples. If there's interest from the community, we can definitely think about including it as a complimentary type to Result.
